crates/sui-config/src/builder.rs

```diff
@@ -93,12 +116,14 @@ impl<R: ::rand::RngCore + ::rand::CryptoRng> ConfigBuilder<R> {
             .map(|(i, validator)| {
                 let name = format!("validator-{i}");
                 let public_key: AuthorityPublicKeyBytes = validator.key_pair.public().into();
```

Comment:
Ideally, in a future PR we have to rename `public_key` to something more meaningful (as we do for `network_key` and `account_key`)